Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -39,14 +34,39 @@ static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAsse
 
             internal static void MoveInputManagerAssetActionsToProjectWideInputActionAsset()
             {
-                var objects = AssetDatabase.LoadAllAssetsAtPath(kAssetPath);
+                var objects = AssetDatabase.LoadAllAssetsAtPath(kAssetPathInputManager);
```

Comment:
I think this needs to use FileUtils.PhysicalPath to be future proof, DM me for further elaboration. See EditorHelpers.cs for how its used conditionally. I have extracted the #ifdef into its own function on a pending PR but its not available in this version.

Reply:
Just change to this?

`#if UNITY_2021_2_OR_NEWER
                var objects = AssetDatabase.LoadAllAssetsAtPath(FileUtil.GetPhysicalPath(kAssetPathInputManager));
#else
                var objects = AssetDatabase.LoadAllAssetsAtPath(kAssetPathInputManager);
#endif
`

Reply:
Do I need to make this change for the other file path too?

```
                        string path = ProjectWideActionsAsset.kDefaultAssetPath;

                        if (File.Exists(path))

```